Formula/ptex.rb

```diff
@@ -1,10 +1,8 @@
 class Ptex < Formula
   desc "Texture mapping system"
   homepage "https://ptex.us/"
-  url "https://github.com/wdas/ptex.git",
-      tag:      "v2.4.1",
-      revision: "93c8bad39a6122c42c9d9d8e29d715bd73a6c575"
-  license "BSD-3-Clause"
+  url "https://github.com/wdas/ptex/archive/refs/tags/v2.4.1.tar.gz"
+  sha256 "664253b84121251fee2961977fe7cf336b71cd846dc235cd0f4e54a0c566084e"
```

Comment:
```suggestion
  sha256 "664253b84121251fee2961977fe7cf336b71cd846dc235cd0f4e54a0c566084e"
  license "BSD-3-Clause"
```
I think you dropped this

Reply:
Indeed I did, surprised the CI didn't yell.